src/lib.rs

```diff
@@ -102,6 +129,14 @@ impl Otel {
         }
     }
 
+    /// Alternative constructor that also returns a shutdown handle.
```

Comment:
I think I agree with @danewalton-msft about not making a breaking change to the existing constructor. And if so, just adding `shutdown_handle()` method that returns the new ShutdownHandle should be enough. We don't need to have a separate new constructor.